.cloudtest.yaml

```diff
@@ -122,6 +122,8 @@ executions:
     kubernetes-env:
       - KUBECONFIG
     run: |
+      kubectl version
+      kubectl get pod -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName --all-namespaces || echo "Kubectl failing when it shouldn't"
```

Comment:
@edwarnicke Unfortunatelly CouldTest for now doesn't support || or any other piping. We just start processes for every line, probable we need to improve it showhow.